src/lib/viewers/controls/settings/Settings.tsx

```diff
@@ -86,13 +75,17 @@ export default function Settings({
         >
             <SettingsContext.Provider value={{ activeMenu, activeRect, setActiveMenu, setActiveRect }}>
```

Comment:
Can we remove `activeRect` from context with this change?

Reply:
Seems like it, will remove